MUXSDKKaltura/MUXSDKKaltura/MUXSDKPlayerBinding.swift

```diff
@@ -383,8 +376,13 @@ public class MUXSDKPlayerBinding: NSObject {
         
         eventVideoData.videoSourceUrl = videoData.url
         
-        if (self.videoData.lastAdvertisedBitrate > 0) {
+        if (self.videoData.lastAdvertisedBitrate > 0 && self.videoData.started) {
             eventVideoData.videoSourceAdvertisedBitrate = NSNumber(value: self.videoData.lastAdvertisedBitrate)
+            
+            if self.videoData.lastDispatchedAdvertisedBitrate != self.videoData.lastAdvertisedBitrate {
```

Comment:
The `self.videoData.hasUpdates = true` from the handleRenditionChange method should be removed as well since we are just adding the logic here this should be the only source of truth. We shouldn't have it duplicated.